packages/compass/src/index.d.ts

```diff
@@ -44,6 +44,11 @@ declare module 'process' {
          * development
          */
         COMPASS_SHOW_YOUR_QUERIES_TAB?: 'true';
+
+        /**
+         * Enable new aggregation pipeline toolbar
+         */
+        SHOW_NEW_AGGREGATION_TOOLBAR?: 'true',
```

Comment:
What about adding the `COMPASS_` prefix like the other flag? ie: `COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR`